recipes/zimg/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "zimg" %}
+{% set version = "3.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sekrit-twc/zimg/archive/release-{{ version }}.tar.gz
+  sha256: 219d1bc6b7fde1355d72c9b406ebd730a4aed9c21da779660f0a4c851243e32f
+
+build:
+  skip: True  # [win]
```

Comment:
are you actually meaning to skip windows? if so, maybe just remove everything that is windows related.

Reply:
I just found `m2-automake` which can run on win64. Gonna add it back to the builds now.

Reply:
Actually, we would prefer that you do not use `m2-` since the `m2` tools generate binaries that are incomaptible with the rest of the windows stuff.

See
https://github.com/conda-forge/ffmpeg-feedstock/blob/main/recipe/meta.yaml#L34

autotools_clang_conda may help

Reply:
All said, I don't really understand the difference between the windows tools too much.

If you do decide to use `m2-` we would have to wait for somebody other than me to review sadly.

Reply:
Really appreciate the help! I was looking for a tool to simply get the job done and `m2-` popped up on the search. The toolchains on Windows look like magic to me. They sometimes depend on each other, while sometimes they are incompatible.

Reply:
I think `m2-` is actually ok. if you can get it to work. i know it is tough.

It is "ok" to just leave the windows as "future work". I would just ask that you delete all the build files.

Reply:
I see. Interestingly, `autotools_clang_conda` is just a wrapper around `m2-` tools, so it seems better to let it do the tough part.